lib/libzfs/libzfs_sendrecv.c

```diff
@@ -277,7 +277,7 @@ cksummer(void *arg)
 		{
 			struct drr_begin *drrb = &drr->drr_u.drr_begin;
 			int fflags;
-			int sz = 0;
+			size_t sz = 0;
```

Comment:
This issue goes deeper than this one change.  There's a fair bit of inconsistency with what types are passed and how this gets eventually stored.  I see all the following types used inconsistently uint64_t, size_t, int, uint32_t and maybe more.  In practice this doesn't cause an issue because this is a small value which fits in all of these types.  Let's mark this one intentional for now and leave it alone, what was the CID I didn't see it in Coverity.